rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -79,6 +81,11 @@ public class BridgeSupportSvpTest {
     private Federation activeFederation;
     private Federation proposedFederation;
 
+    private Sha256Hash svpFundTransactionHashUnsigned;
+    private BtcTransaction svpFundTransaction;
+    private Sha256Hash svpSpendTransactionHashUnsigned;
+    private BtcTransaction svpSpendTransaction;
```

Comment:
Maybe you can have all these be default constants to be reused by the tests.

Reply:
have in mind that in some cases we are testing the tx creations , so we cannot previously define them.